File: visualBasic/Ice/callback/CallbackReceiverI.vb
' **********************************************************************
'
' Copyright (c) 2003-2016 ZeroC, Inc. All rights reserved.
'
' **********************************************************************

Imports Demo

Public NotInheritable Class CallbackReceiverI
    Inherits CallbackReceiverDisp_

    Public Overloads Overrides Sub callback(ByVal current As Ice.Current)
        System.Console.Out.WriteLine("received callback")
    End Sub

End Class

File: java/Ice/callback/CallbackReceiverI.java
// **********************************************************************
//
// Copyright (c) 2003-2016 ZeroC, Inc. All rights reserved.
//
// **********************************************************************

import Demo.*;

public final class CallbackReceiverI implements CallbackReceiver
{
    @Override
    public void callback(com.zeroc.Ice.Current current)
    {
        System.out.println("received callback");
    }
}
